mirrord-schema.json

```diff
@@ -668,21 +668,18 @@
               "$ref": "#/definitions/PodTarget"
             }
           ]
+        },
+        {
+          "description": "Run without a target. An independent agent pod will be spawned in an arbitrary node.",
```

Comment:
Good question, I guess it would not have any effect on functionality, right? But maybe some users would like to choose the node for cloud organization reasons, idk. We could also add it in a separate PR, or open an issue and prioritize it first, to deliver this feature asap. wdyt?

Reply:
I don't need we need it unless someone asks for it.